src/Diagnostics.DataProviders/DataProviderLogDecorator.cs

```diff
@@ -184,6 +184,11 @@ public Task<T> MakeHttpGetRequestWithFullPath<T>(string fullPath, string querySt
             return MakeDependencyCall(null, _geomasterDataProvider.MakeHttpGetRequestWithFullPath<T>(fullPath, queryString, apiVersion));
         }
 
+        public Task<string> GetContainerLogs(string subscriptionId, string resourceGroupName, string name)
```

Comment:
I am guessing its linux container (docker or ACR).  Remember there is support for Windows Containers now also in antares. 
If this method is only targeting Linux, maybe call it 'GetLinuxContainerLogs' ?

Reply:
good point, changed it to 'GetLinuxContainerLogs'